CommonMark/CommonMarkAdditionalFeatures.cs

```diff
@@ -19,13 +19,27 @@ public enum CommonMarkAdditionalFeatures
         /// The parser will recognize syntax <c>~~foo~~</c> that will be rendered as <c>&lt;del&gt;foo&lt;/del&gt;</c>.
         /// </summary>
         StrikethroughTilde = 1,
-
-        /// <summary>
-        /// The parser will recognize syntax <c>[foo]</c>, which will be encoded in a separate AST node that the host application may evaluate as desired.
+		
+		/// <summary>
+		/// The parser will recognize syntax <c>[foo]</c>, which will be encoded in a separate AST node that the host application may evaluate as desired.
         /// </summary>
         PlaceholderBracket = 2,
-
+		
         /// <summary>
+        /// The parser will recognize
+        /// 
+        /// First Header  | Second Header
```

Comment:
add `<code>` around the example (I *suppose* that is the correct tag as otherwise the intellisense could format the example).